package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "iggy",
+  "description": "Official nodejs iggy.rs binary client",
   "type": "module",
-  "version": "1.3.0",
-  "description": "official nodejs iggy.rs binary client",
+  "version": "1.3.5",
```

Comment:
package.json version is edited by semantic release bot on merge, please do not edit manually.
when we settle the token issue first version will be resetted to 1.0.0 if we don't add some "current" version git tag before semantic release first run.